src/services/can.ts

```diff
@@ -1,6 +1,9 @@
 import { InjectionKey, ref } from 'vue'
 import { MaybeRef } from '@/types/reactivity'
 
+export const featureFlags = [] as const
```

Comment:
to be consistent with `workspacePermissions` should these all be `workspaceFeatureFlags`?